assets/css/startertheme.css

```diff
@@ -121,6 +121,10 @@
 
 
 /* About Us */
+.mt-more {
+  margin-top: 93.5rem;
+}
```

Comment:
Why do you need such a large margin? This might end up breaking on smaller screens. Could you attach a screenshot of what this is helping you achieve?

Reply:
fixed margin-top